### ISSUE TITLE
Grafana 5.1.3 

### ISSUE THREAD

**Issue by ryadav88** [mesosphere/universe#1911]
PR 1503 
@deric - `What's the motivation behind these changes? It appears as a regression to me. Why add java binary? Health check path /v1/health returns 404.`

The package is now built with https://github.com/mesosphere/dcos-commons/
It uses a readiness check instead. Java binary is for the scheduler. 
Also, I do not see a 404. ?

**Comment by deric** [comment]
How do I update such package? `/v1/health` is scheduler's endpoint? Also these ENV variables were removed in #1903:
```
        "GF_SECURITY_ADMIN_PASSWORD": "{{grafana.admin_password}}",
        "GF_INSTALL_PLUGINS": "{{grafana.plugins}}"
```

**Comment by adiep-cdpq** [comment]
How am I supposed to install package with the json config file?

**Comment by deric** [comment]
@ryadav88 Yes, I know `dcos-commons`. However I have no idea where to find the source code for Grafana framework. e.g. is `svc.yml` https://ecosystem-repo.s3.amazonaws.com/grafana/artifacts/5.5.0-5.1.3/svc.yml versioned in some git repo?

**Comment by a-nldisr** [comment]
Why not use the docker container from the grafana community and utilize this? Previous DCOS installation was using docker containers from the community, when i adjusted this to the latest Grafana version 6+ it was working.
